WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingsViewController.swift

```diff
@@ -364,11 +474,15 @@ open class NotificationSettingsViewController: UIViewController {
     fileprivate let loadMoreRowIndex                = 3
     fileprivate let loadMoreRowCount                = 4
 
+
     // MARK: - Private Properties
-    fileprivate var groupedSettings: [[NotificationSettings]]?
+
+    fileprivate var groupedSettings: [Section: [NotificationSettings]]?
     fileprivate var displayMoreWasAccepted          = false
```

Comment:
You might consider refactoring how `displayMoreWasAccepted` works, or maybe use a different flag for the followed sites list rather than sharing a single flag between both lists.  When there is multiple blogs in both lists it looks like having a single flag is going to end up expanding both lists (assuming there is no exception).